components/docs-chef-io/content/automate/ha_inplace_migration.md

```diff
@@ -0,0 +1,173 @@
+    rename /hab dir to something else like /hab-old
+
+
+5. Unload services from each of Elasticsearch Nodes
+   ```cmd
+    sudo hab svc unload chef/automate-backend-elasticsidecar
+    sudo hab svc unload chef/automate-backend-elasticsearch
+    sudo hab svc unload chef/automate-backend-journalbeat
+    sudo hab svc unload chef/automate-backend-metricbeat
+    sudo hab svc unload chef/automate-backend-curator
+    sudo hab svc unload chef/automate-backend-kibana
+   ```
+   check status use ``` hab svc status ``` none of services should be running
+   then stop habitat superviser with command ``` systemctl stop hab-sup ```
+   rename /hab dir to something else like /hab-old
+
+
+6. In bastion host take copy of your current workspace and keep it safe for while
+7. remove or rename /hab dir in bastion host
+
+## Installing Latest Automate HA
+Follow Automate HA installation documentations Click [here](/automate/ha_onprim_deployment_procedure/) to know more
+
+in config.toml give same IPs and backup config as it was in a2ha.rb file
+
+## Restore Backup 
+
+once deployment is successful, 
+Now proceed with restoring the backup in Automate HA, 
+login to one of automate node and take current_config,toml file
+``` sudo chef-automate config show > current_config.toml ```
+
+add the following config to current_config.toml
+
+```cmd
+[global.v1.external.opensearch.auth.basic_auth] 
+    username = "admin"
+    password = "admin"
+```
+
+and
+
+```cmd
+[global.v1.external.opensearch.backup.fs]
+    path = "/mnt/automate_backups/elasticsearch"
+```
+
+to restore use below command
+```cmd
+sudo chef-automate backup restore /mnt/automate_backups/backups/20210622065515/ --patch-config current_config.toml --airgap-bundle /var/tmp/frontend-4.x.y.aib --skip-preflight
+```
+
+Click [here](/automate/ha_backup_restore_file_system/) to know more
+
+{{< note >}}
+
+- After the restore command is successfully executed. If we run the `chef-automate config show`, we can see that both ElasticSearch and OpenSearch config are part of Automate Config. We can keep both the config; it won't impact the functionality. After restoring Automate HA, talk to OpenSearch.
+
+OR
+
+- We can remove the ElasticSearch config from the automate. To do that, redirect the applied config to the file and set the config again.
+
+```bash
+chef-automate config show > applied_config.toml
+```
+
+Modify `applied_config.toml`, remove elastic search config, and set the config. Set `applied_config.toml` on all the frontend nodes manually. As the removal of config is not supported from the bastion. Use the below command to set the config manually.
+
+```bash
+chef-automate config set applied_config.toml
+```
+  
+{{< /note >}}
+
+{{< note >}}
+In case of ``` backup_config = "file_system" ``` had been provided in config.toml of Automate HA deployment, then need to patch below opensearch config from bastion before starting restore..
```

Comment:
```suggestion
In case ``` backup_config = "file_system" ``` had been provided in config.toml of Automate HA deployment, then we need to patch the below OpenSearch config from bastion before starting the restore.
```